File: loan-repayment-java/src/main/java/com/loan/repayment/entity/RepaymentPlanResponse.java
package com.loan.repayment.entity;

import java.util.List;

import io.swagger.annotations.ApiModelProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;

/**
 * 
 * @author Ramachandra
 *
 */
@Builder(toBuilder = true)
@NoArgsConstructor
@AllArgsConstructor
@Getter
@EqualsAndHashCode
public class RepaymentPlanResponse {

    /**
     * 
     * Borrower Payments.
     */
    @ApiModelProperty(notes = "Borrower Payments")
    private List<BorrowerPayment> borrowerPayments;

}

File: loan-repayment-kotlin/src/main/kotlin/com/loan/repayment/entity/RepaymentPlanResponse.kt
package com.loan.repayment.entity

import com.fasterxml.jackson.annotation.JsonProperty
import io.swagger.annotations.ApiModelProperty

/**
 *
 * @author Rama chandra
 */
data class RepaymentPlanResponse(
        @ApiModelProperty(notes = "Borrower Payments")
        @JsonProperty("borrowerPayments")
        var borrowerPayments: List<BorrowerPayment>
)